modules/tasks/taskPayment.js

```diff
@@ -27,14 +27,7 @@ module.exports = Promise.method(function taskPayment (paymentParams) {
       }).then(async assign => {
         const user = assign.dataValues.User.dataValues
 
-        let transferGroup = null
-
-        const order = await models.Order.findOne({ where: { TaskId: task.id } })
-        const coupon = await models.Coupon.findOne({ where: { id: order.couponId } })
-
-        if (!coupon || (coupon && coupon.amount < 100)) {
-          transferGroup = task.transfer_group ? task.transfer_group : `task_${task.id}`
-        }
+        const transferGroup = task.transfer_group ? task.transfer_group : `task_${task.id}`
```

Comment:
@VictorAlessander with this solution the error maybe will persist in some cases.

For coupon with 100% the transfer group shouldn't be sent, so the way it before made sense, the error is that you're sending empty, and the rule here is it will not send the parameter if is empty.

So maybe you initialize the transfer params object first like:

```
let transferData = {
          amount: centavosAmount * 0.92, // 8% base fee
          currency: 'usd',
          destination: dest,
          source_type: 'card'
        }
``` 

And on the if clause you may add:

`transferData.transferGroup = ...`

So it will only define `transferGroup` inside the condition